### ISSUE TITLE
[Screen Reader-Help-Projects] Role and Title is not defined property for Projects screen

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2540]
**User Experience:**
If any control have incorrect role defined then the person with visual related disability will find it difficult to interpret the proper fields information. This will hinder him to access all the controls of the page efficiently and also not compatible with AT.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Prerequisites:** Turn on Narrator

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section
4-Verify, If role property is defined correct for projects screen or not.

**Actual Result**
Role property is defined incorrect for project screen. Role is defined as "Complementary". Title/name attribute is missing for iframe.

**Expected Result**
There is no need to defined role for iframe and  Title/name attribute should be defined so that it will inform user about the project screen.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested fix**

Using ARIA role to expose the control type of user interface component
e.g.
< div role="toolbar" tabindex="0" id="customToolbar" >
< img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home" >
Please Refer attachment for more details
![mas4 1 2_role is defined incorrect for project screen](https://user-images.githubusercontent.com/30459830/28661921-fe768f88-72d5-11e7-8f13-039982cb8522.png)

[Projects_Narrator reading role incorrect.zip](https://github.com/Microsoft/pxt/files/1182460/Projects_Narrator.reading.role.incorrect.zip)

**Comment by v-mesala** [comment]
Bug is verified and fixed.Hence closed the bug.
